src/Jackett.Common/Definitions/hdonly.yml

```diff
@@ -88,177 +97,162 @@ login:
-        - name: replace
-          args: [".VFQ.MULTI", ".MULTI.VFQ"]
-        - name: replace
-          args: ["[Film]", ""]
-        - name: replace
-          args: ["[Dessin animé]", ""]
-        - name: replace
-          args: ["[Film d'animation]", ""]
-        - name: replace
-          args: ["[Concert]", ""]
-        - name: replace
-          args: ["[Documentaire]", ""]
-        - name: replace
-          args: ["[Court-métrage]", ""]
-        - name: replace
-          args: ["[Clip]", ""]
-        - name: replace
-          args: ["[Démonstration]", ""]
-        - name: replace
-          args: ["[Bonus de BD]", ""]
-        - name: replace
-          args: ["[Autre]", ""]
-        - name: replace
-          args: ["[Série Animée]", ""]
+          args: ["mHD", "mHD.BluRay.Rip"]
+    _encoding:
+      selector: encoding
+    _release_group:
+      selector: team
+      optional: true
+    _media:
+      selector: media
+    _vff:
+      selector: vff
+    _vfq:
+      selector: vfq
+    _vfi:
+      selector: vfi
+    _vf:
+      selector: vf
+    _vfo:
+      selector: vfo
+    _hdr:
+      selector: hdr
+    _hdr10plus:
+      selector: hdr10plus
+    _dovi:
+      selector: dovi
+    title_optional:
+      selector: group.groupName
+      filters:
+        - name: append
+          args: ".{{ .Result.year }}.{{ .Result._episode_info }}.{{ .Result._format }}.{{ .Result._encoding }}.{{ .Result._media }}{{ if eq .Result._hdr \"True\" }}.HDR{{ else }}{{ end }}.{{ if eq .Result._hdr10plus \"True\" }}.HDR10+{{ else }}{{ end }}.{{ if eq .Result._dovi \"True\" }}.DoVi{{ else }}{{ end }}"
+        - name: append
+          args: "{{ if or (eq .Result._vff \"True\") (eq .Result._vfq \"True\") }}.MULTI{{ else }}{{ end }}{{ if or (eq .Result._vfi \"True\") (eq .Result._vf \"True\") }}.MULTI{{ else }}{{ end }}"
         - name: replace
-          args: ["[Série]", ""]
-        - name: re_replace
-          args: ["\\s+", "."]
-        - name: re_replace
-          args: ["\\.*([\\-:,])\\.*", "$1"]
+          args: [".MULTI.MULTI", ".MULTI"]
+        - name: append
+          args: "{{ if eq .Result._vfo \"True\" }}.FRENCH{{ else }}{{ end }}"
         - name: re_replace
-          args: ["\\[(\\d{4})\\]", "$1"]
+          args: ["\\.+", "."]
+        - name: trim
         - name: trim
+          args: .
         - name: append
           args: "{{ if .Result._release_group }}-{{ .Result._release_group }}{{ else }}{{ end }}"
+    _filename:
+      selector: "fileName"
+      optional: true
+      filters:
+        - name: replace
+          args: [".mkv", ""]
+        - name: append
+          args: " {{ if and (.Config.add_hybrid_features_to_filename) (eq .Result._hdr \"True\") }}[HDR]{{ else }}{{ end }}{{ if and (.Config.add_hybrid_features_to_filename) (eq .Result._hdr10plus \"True\") }}[HDR10+]{{ else }}{{ end }}{{ if and (.Config.add_hybrid_features_to_filename) (eq .Result._dovi \"True\") }}[DoVi]{{ else }}{{ end }}"
```

Comment:
Move to `title_phase1`. Currently ending up with titles like:
`Massacre à la tronçonneuse.1974.HEVC.2160p.Blu-ray 4K.HDR.DoVi.MULTi.FRENCH-ONLY`
but with the description ending `[HDR][DoVi]`.

Reply:
I'm lost, if we're adding `[HDR][DoVi]` to:
`The.Texas.Chain.Saw.Massacre.1974.2160p.FRA.UHD.Blu-ray.DV.HDR10.HEVC.TrueHD.7.1-ONLY`

why not add `[HDR][DoVi]` to:
`Massacre à la tronçonneuse.1974.HEVC.2160p.Blu-ray 4K.HDR.DoVi.MULTi-ONLY`
?

Reply:
...I'm clearly blind 😛